neqo-transport/src/streams.rs

```diff
@@ -154,6 +158,8 @@ impl Streams {
                 stats.stream += 1;
                 if let (_, Some(rs)) = self.obtain_stream(*stream_id)? {
                     rs.inbound_stream_frame(*fin, *offset, data)?;
+                } else {
+                    return Err(Error::StreamStateError);
                 }
```

Comment:
Are these better as let/else?

```suggestion
                let (_, Some(rs)) = self.obtain_stream(*stream_id)? else {
                    return Err(Error::StreamStateError);                    
                };
                rs.inbound_stream_frame(*fin, *offset, data)?;
```

Reply:
Or...

```suggestion
                let rs = self.obtain_stream(*stream_id)?.1.ok_or(Error::StreamStateError)?;
                rs.inbound_stream_frame(*fin, *offset, data)?;
```